linux-pull-run.sh

```diff
@@ -53,14 +93,65 @@ while [[ "$1" =~ ^- && ! "$1" == "--" ]]; do case $1 in
 	--tag=* )
 		# Make sure a tag was not already specified
 		_enforce_one_tag
-		
+
 		# Get the characters immediately after the equals sign in the CLA
 		tag="${1##--tag=}"
 
 		# Check that a tag was provided
 		# Strip the equals sign and anything after it from the CLA to make the field "--tag"
 		_validate_arg "${1%%=*}" $tag
 		;;
+	-d | --dist-tag )
+		# Make sure an engineering-data-distributor tag was not already specified
+		_enforce_one_distributor_tag
+
+		# Get the engineering-data-distributor tag specified after the -d or --dist-tag flag
+		dist_tag=$2
+
+		# Check that a tag was provided
+		# Keep only characters before a hyphen from the CLA after -d | --dist-tag, effectively ignoring that CLA if it is a
+		# flag
+		_validate_arg $1 "${dist_tag%%-*}"
+		shift
+		;;
+	--dist-tag=* )
+		# Make sure an engineering-data-distributor tag was not already specified
+		_enforce_one_distributor_tag
+
+		# Get the characters immediately after the equals sign in the CLA
+		dist_tag="${1##--dist-tag=}"
+
+		# Check that a tag was provided
+		# Strip the equals sign and anything after it from the CLA to make the field "--dist-tag"
+		_validate_arg "${1%%=*}" $dist_tag
+		;;
+	-c | --config )
+		# Make sure a configuration was not already specified
+		_enforce_one_configuration
+
+		# Get the configuration specified after the -c or --config flag
+		config=$2
+
+		# Check that a valid configuration was provided
+		if [[ "$config" != @(competition|dev|individual) ]]; then
```

Comment:
```suggestion
		if [[ "$config" != competition && "$config" != dev && "$config" != individual ]]; then
```
I'm getting a syntax error here running on my Mac, changing to this seems to work

Reply:
Fixed